Dockerfile

```diff
@@ -13,7 +13,7 @@ COPY pyproject.toml poetry.lock ./
 RUN mkdir /static
 
 RUN apt-get update \
-    && apt-get install -y libpq-dev gcc \
+    && apt-get install -y libpq-dev gcc libpcre3 libpcre3-dev \
```

Comment:
We need `pcre` support for the part of the app.ini file that says `route = ^/healthz donotlog:`